Riva0110/Vocab-Garden
# Feature/search vocab

### Description

- Contect to Vocab API
- Show Vocab details, including audio, definitions, examples, synonyms
- Add keyword search

### Test Plan

- Try different keywords
- Check if audio can play successfully
